apps/plasmo/src/components/Calendar/CalendarMonthYear.tsx

```
@@ -0,0 +1,179 @@
+      const spaceBelow = windowHeight - buttonRect.bottom
+      setMaxHeight(Math.min(250, spaceBelow - 10))
+    }
+  }, [isOpen])
+
+  const selectedOption = options.find((option) => option.value === value)
+
+  return (
+    <div className="plasmo-relative" ref={dropdownRef}>
+      <button
+        onClick={() => setIsOpen(!isOpen)}
+        className="plasmo-flex plasmo-items-center plasmo-justify-between plasmo-w-full plasmo-px-2 plasmo-py-1.5 plasmo-text-xs plasmo-text-gray-200 plasmo-bg-gray-800 plasmo-border plasmo-border-gray-700 plasmo-rounded-md hover:plasmo-bg-gray-700 focus:plasmo-outline-none focus:plasmo-ring-2 plasmo-ring-blue-500">
```

Biome

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)